tools/webots.py

```diff
@@ -3,22 +3,31 @@
 import glob
 import multiprocessing
 import os
+import random
 import shutil
 import subprocess
 import sys
 import textwrap
+from datetime import datetime
 from pathlib import Path
+from subprocess import DEVNULL
 
+from honcho.manager import Manager
 from termcolor import cprint
 
 import b
 from utility.dockerise import platform
 
+random.seed(datetime.now())
+
 # The docker image details for Robocup
 ROBOCUP_IMAGE_NAME = "robocup-vhsc-nubots"  # Provided by the TC and shouldn't be changed
 ROBOCUP_IMAGE_TAG = "robocup2021"  # Submitted in our team_config.json, shouldn't be changed here unless changed there
 ROBOCUP_IMAGE_REGISTRY = "079967072104.dkr.ecr.us-east-2.amazonaws.com/robocup-vhsc-nubots"  # Provided by the TC
 
+# Generate 4 digit random int to act as a unique identifier for this game
+GAME_IDENTIFIER = str(random.randint(1000, 9999))
```

Comment:
How about something [like this](https://stackoverflow.com/a/27302466/4795763) instead? Might be overkill?

Reply:
I think that is likely overkill for now, If we do ever run into issues we can definietly add something like that though.